wamr-compiler/main.c

```diff
@@ -68,6 +68,12 @@ print_help()
     printf("  --xip                     A shorthand of --enalbe-indirect-mode --disable-llvm-intrinsics\n");
     printf("  --enable-indirect-mode    Enalbe call function through symbol table but not direct call\n");
     printf("  --disable-llvm-intrinsics Disable the LLVM built-in intrinsics\n");
+    printf("  --enable-builtin-intrinsics=<i64.common,f64.common>\n");
+    printf("                            Enable the specified built-in intrinsics, it will override the default settings.\n");
```

Comment:
Seems a little too long this line, can we put `settings.` to next line?

Reply:
OK